recipe/magento2.php

```diff
@@ -181,10 +185,19 @@
 desc('Deploy your project');
 task('deploy', [
     'deploy:prepare',
-    'deploy:vendors',
     'deploy:clear_paths',
     'deploy:magento',
     'deploy:publish',
 ]);
 
+if (get('push_strategy') === false) {
```

Comment:
But this code executed right away. How to override it in users recipe?

Reply:
Good point. Using the `disabled()` method this isn't needed because we can do this in our custom recipe;

```
task('deploy:vendors')->disabled();
task('deploy:update_code')->disabled();
task('magento:compile')->disabled();
task('magento:deploy:assets')->disabled();
```